app/cdap/components/ImportDataset/constants.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+export const IMPORT_DATASET = 'Import Dataset';
+export const WRANGLE = 'Wrangle';
+export const DRAG_AND_DROP_TEXT_LINE_1 = 'Drag and drop a file to be uploaded ';
+export const DRAG_AND_DROP_TEXT_LINE_2 = 'or Click here to select a file from your computer';
+export const MAX_SIZE_TEXT = 'Max file size: 10 MB, Supported file formats: CSV, JSON';
```

Comment:
use translation

Reply:
Deleted the constant file and the texts are already in text-en.yaml file.